src/notes.txt

```diff
@@ -0,0 +1,194 @@
+Issue addesses:

```

Comment:
Is this like a wiki file?

If so, can you change to .md please?

Reply:
This file does not need to be here. I'll remove it. This was a file used for my own reference.